db/db.go

```diff
@@ -0,0 +1,35 @@
+package db
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var dbh *sql.DB
+var err error
```

Comment:
It was weird to me to. I added it so that every call to `db.Get()` returned a consistant DB handle and error. Before the error was global the first call would return a non-nil error and subsequent calls would return a broken DB handle and nil error b/c it didn't have access to the error from the previous `sql.Open` and `sql.Ping`. I believe I've address this threw the refactor of the module.